### PR TITLE
Fixing issues when bumping @types/react to 18.0.0

### PR DESCRIPTION
**Pull request by tbwiss**
# Description

Renovate bumped the version of @react/types to 18.0.0 in https://github.com/neo4j/graphql/pull/1242. It resolved in a few typing errors which are addressed in this PR.
Added an entry in the GraphQL Toolbox feedback list to replace the  [react-grid-resizable](https://www.npmjs.com/package/react-grid-resizable) dependency.

### CODE REVIEW

**Comment by neo4j-team-graphql** [comment]
Looks like you've updated the UI!

Check out your changes at
 http://neo4j-graphql-toolbox-stage-1244.surge.sh (=> for local development; use for instance [Neo4j Desktop](https://neo4j.com/download/))
or
 https://neo4j-graphql-toolbox-stage-1244.surge.sh (=> use to connect to an [Aura Free](https://neo4j.com/cloud/aura-free/) dbms)

**Review by darrellwarde — DISMISSED** [review]
Looks good, don't know what's going on with the failing package tests though!

```
Error: ../../../node_modules/@types/styled-components/index.d.ts(21,16): error TS2694: Namespace 'React' has no exported member 'ReactType'.
```

File: packages/package-tests/typescript/tsconfig.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
     "baseUrl": "./",
     "sourceRoot": "./",
     "compilerOptions": {
+        "skipLibCheck": true,
```
Comment (tbwiss) [inline packages/package-tests/typescript/tsconfig.json:6]: @darrellwarde The errors that caused the typescript package-test to fail all occurred in the `node_modules` folder. Using `skipLibCheck` ([docs](https://www.typescriptlang.org/tsconfig#skipLibCheck)) solves the issue. Is there anything against this approach you can think of?

**Comment by darrellwarde** [comment]
Just converting this to a draft to make it clear this won't be done right now!

**Comment by tbwiss** [comment]
Closing this PR as this has been resolved in https://github.com/neo4j/graphql/pull/1282

**Pull request closed without merging**